resources/views/components/layout.blade.php

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>HZ HBO-ICT</title>
+
+    {{-- Compiled assets --}}
+    @vite(['resources/sass/app.scss', 'resources/js/app.js'])
+
+    <style>
+        <?php include 'C:\Users\rosie\Documents\HBO-ICT\year 1\quarter 3\Frameworks Project 1\projects\portfolio-app\resources\css\main.css'; ?>
```

Comment:
Hard coded files to include is a very bad practice. Laravel has a better way to solve this.